jhu-idc/idc-isle-dc
# Adds Metadata Export Test

This adds new tests for metadata export. These tests test out the export and compares results for each field on each item to ensure that the data is exported correctly. 
 
Tests: 
 - Single Item Metadata Export
 - Single Collection Metadata Export
 - Search Results - Repository Item Export
 - Search Results - Collection Export
 - Round Trip test - 
     - this test will export a repository item, change a few fields, create a new csv file, and then ingest the new csv file. It will then export the node again and test values, ensuring that the few fields did actually change. It does look at the node's id to ensure that the ingest was an update and that it didn't create a new object. 

Resolves https://github.com/jhu-idc/iDC-general/issues/329
Resolves https://github.com/jhu-idc/iDC-general/issues/327
Resolves https://github.com/jhu-idc/iDC-general/issues/330
